pallets/schemas/src/lib.rs

```diff
@@ -184,16 +188,24 @@ pub mod pallet {
 	pub(super) type CurrentSchemaIdentifierMaximum<T: Config> =
 		StorageValue<_, SchemaId, ValueQuery>;
 
+	/// Storage for message schema info struct data
+	/// - Key: Schema Id
+	/// - Value: [`SchemaInfo`](SchemaInfo)
+	#[pallet::storage]
+	#[pallet::getter(fn get_schema_info)]
+	pub(super) type SchemaInfos<T: Config> =
+		StorageMap<_, Twox64Concat, SchemaId, SchemaInfo, OptionQuery>;
+
 	/// Storage for message schema struct data
 	/// - Key: Schema Id
-	/// - Value: [`Schema`](Schema)
+	/// - Value: [`BoundedVec`](BoundedVec<T::SchemaModelMaxBytesBoundedVecLimit>)
 	#[pallet::storage]
-	#[pallet::getter(fn get_schema)]
-	pub(super) type Schemas<T: Config> = StorageMap<
+	#[pallet::getter(fn get_schema_payload)]
+	pub(super) type SchemaPayloads<T: Config> = StorageMap<
```

Comment:
minor: Ok. I think we can drop the `s` here and call it `SchemaPayload`.

Reply:
I think either we need to keep all of them plural or singular. Based on what we have in some pallets we have singulars like in Msa or handles and in other places plural like in Messages and time-release